operator-common/pom.xml

```diff
@@ -104,6 +104,7 @@
         <dependency>
             <groupId>com.github.spotbugs</groupId>
             <artifactId>spotbugs-annotations</artifactId>
+            <scope>provided</scope>
```

Comment:
Is it not sufficient to set the scope in the `dependencyManagement` of the root pom?

Reply:
Let me check ...

Reply:
🤦‍♂️... you are of course right. I updated the PR to set it onyl in dependnecy management.